### PR TITLE
Move to CMake targets

### PR DESCRIPTION
**Pull request by awegrzyn**

### CODE REVIEW

**Comment by awegrzyn** [comment]
@sy-c Travis fails due to:
```
/home/travis/build/AliceO2Group/Common/test/TestSystem.cxx(65): error: in "TestGetFileSystemType": check System::getFileSystemType("/dev") == "devtmpfs" has failed
/home/travis/build/AliceO2Group/Common/test/TestSystem.cxx(66): error: in "TestGetFileSystemType": check System::getFileSystemType("/dev/shm") == "tmpfs" has failed
```
Do you think you could fix it? Otherwise I'll just disable the test.

**Comment by awegrzyn** [comment]
These changes do not affect any package using `FindCommon.cmake` macro

**Comment by awegrzyn** [comment]
This PR obsoletes https://github.com/AliceO2Group/Common/pull/15

**Comment by sy-c** [comment]
looks good, I just need a bit of time to check a few possible extra cleanup (build_standalone and tcl)

**Review by sy-c — APPROVED** [review]
ok, all good for me
tried with success to rebuild infologger with this new branch

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,153 +1,180 @@
...
+if(NOT CMAKE_BUILD_TYPE)
+  set(
+    CMAKE_BUILD_TYPE "RelWithDebInfo"
+    CACHE
+      STRING
+      "Choose the type of build, options are: Debug Release RelWithDebInfo MinSizeRel Coverage."
+    FORCE)
+endif(NOT CMAKE_BUILD_TYPE)
 
 # Add compiler flags for warnings and (more importantly) fPIC and debug symbols
```
Comment (Barthelemy) [inline CMakeLists.txt]: The comment does not make sense any more

File: include/Common/Version.h.in [hunk 2]
```diff
@@ -13,13 +13,13 @@
 namespace AliceO2 {
 namespace Common {
 /// The current major version.
-#define HELLO_VERSION_MAJOR @VERSION_MAJOR@
+#define HELLO_VERSION_MAJOR @PROJECT_VERSION_MAJOR@
```
Comment (Barthelemy) [inline include/Common/Version.h.in]: Maybe rename HELLO to COMMON ?

**Review by Barthelemy — CHANGES_REQUESTED** [review]
just minor changes

**Review by Barthelemy — APPROVED** [review]

**Pull request merged**